File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/CargoResponse.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.CargoResponseSensors
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Contains the current cargo status of a sensor.
 * @param groupId 
 * @param sensors 
 */
data class CargoResponse (

        @JsonProperty("groupId") val groupId: Long? = null,

        @JsonProperty("sensors") val sensors: List<CargoResponseSensors>? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/CargoResponse.scala

package org.openapitools.client.model


case class CargoResponse (
    _groupId: Option[Long],
    _sensors: Option[List[CargoResponseSensors]]
)
object CargoResponse {
    def toStringBody(var_groupId: Object, var_sensors: Object) =
        s"""
        | {
        | "groupId":$var_groupId,"sensors":$var_sensors
        | }
        """.stripMargin
}
